packages/fiori/src/i18n/messagebundle.properties

```diff
@@ -50,32 +50,25 @@ NOTIFICATION_LIST_ITEM_READ=read
 #XTXT: Text for the NotificationListGroupItem "unread" state
 NOTIFICATION_LIST_ITEM_UNREAD=unread
 
-#XTXT: Text for the NotificationListItem status
-NOTIFICATION_LIST_ITEM_INFORMATION_STATUS_TXT=Status Information
-NOTIFICATION_LIST_ITEM_POSITIVE_STATUS_TXT=Status Positive
-NOTIFICATION_LIST_ITEM_NEGATIVE_STATUS_TXT=Status Negative
-NOTIFICATION_LIST_ITEM_CRITICAL_STATUS_TXT=Status Critical
-
-#XTXT: Text for the NotificationListItem Importance label
-NOTIFICATION_LIST_ITEM_IMPORTANT_TXT=Important
+#XTXT: Text for the NotificationListItem priority
+NOTIFICATION_LIST_ITEM_HIGH_PRIORITY_TXT=High Priority
+NOTIFICATION_LIST_ITEM_MEDIUM_PRIORITY_TXT=Medium Priority
+NOTIFICATION_LIST_ITEM_LOW_PRIORITY_TXT=Low Priority
 
 #XTXT: Text for the NotificationListGroupItem
 NOTIFICATION_LIST_GROUP_ITEM_TXT=Notification group
 
 #XTXT: Text for the NotificationListGroupItem counter
 NOTIFICATION_LIST_GROUP_ITEM_COUNTER_TXT=Counter
 
-#XBUT: accessible name for 'Toggle' button in the NotificationListGroupItem
-NOTIFICATION_LIST_GROUP_ITEM_TOGGLE_BTN_COLLAPSE_TITLE=Collapse
+#XBUT: accessible name for 'Close All' button in the NotificationListGroupItem
```

Comment:
Seems like these texts weren't changed correctly during the merge conflict resolving.